### PR TITLE
Add fast exposure property in CCD_STREAMING

### PR DESCRIPTION
**Pull request by naheedsa**

### CODE REVIEW

File: drivers/README.md [hunk 1]
```diff
@@ -218,6 +218,8 @@ Some CCD drivers support streaming and recording of video streams. All the prope
 |                    |        | RECORD_DURATION_ON       | Start recording until the duration set in `RECORD_OPTIONS` has elapsed.               |
 |                    |        | RECORD_FRAME_ON          | Start recording until the number of frames set in `RECORD_OPTIONS` has been captured. |
 |                    |        | RECORD_OFF               | Stops recording.                                                                      |
+| Fast Exposure      | Switch | INDI_ENABLED             |  Fast Exposure is used to enable camera to immediately begin capturing the next frames|
+|                    |        | INDI_DISABLED            |                                                                                       |
```
Comment (knro) [inline drivers/README.md]: Why did you remove this? We need to keep it and also add exposure counter. So we have two:
1. CCD_FAST_TOGGLE
2. CCD_FAST_COUNT

Please make sure both are added in standard properties

Comment (naheedsa) [reply drivers/README.md]: Removed the description, it was not for Fast Exposure.
And for the Exposure counter, do we need description and Doc?

Comment (knro) [reply drivers/README.md]: I put that in , but you can add this summary: Number of fast exposure captured to take once capture begins

Comment (naheedsa) [reply drivers/README.md]: sure, updated it

File: drivers/README.md [hunk 2]
```diff
@@ -218,6 +218,8 @@ Some CCD drivers support streaming and recording of video streams. All the prope
 |                    |        | RECORD_DURATION_ON       | Start recording until the duration set in `RECORD_OPTIONS` has elapsed.               |
 |                    |        | RECORD_FRAME_ON          | Start recording until the number of frames set in `RECORD_OPTIONS` has been captured. |
 |                    |        | RECORD_OFF               | Stops recording.                                                                      |
+| Fast Exposure      | Switch | CCD_FAST_TOGGLE          | Fast Exposure is used to enable camera to immediately begin capturing the next frames.|
```
Comment (knro) [inline drivers/README.md]: That's not correct, the property name is CCD_FAST_TOGGLE, it has two element INDI_ENABLED and INDI_DISABLED. Check the code and revise the description.
